apps/mobile/src/features/Signer/components/SignerView.tsx

```diff
@@ -82,13 +92,17 @@ export const SignerView = ({
           </XStack>
         </Container>
       </ScrollView>
-      {!editMode && (
+      <KeyboardAvoidingView behavior="padding" keyboardVerticalOffset={top + CUSTOM_VERTICAL_OFFSET}>
```

Comment:
why the custom vertical offset here? I think that you need to add top + bottom. I think the bottom inset is around 50.